packages/web-pkg/src/components/ViewOptions.vue

```diff
@@ -298,7 +298,7 @@ export default defineComponent({
 .oc-range {
   -webkit-appearance: none;
   border-radius: 0.3rem;
-  background: var(--oc-role-secondary-container);
+  background: var(--oc-role-surface-container-high);
```

Comment:
According to material design documentation we could (or should?) use `tertiary-container` for the slider and also for OcSwitch enabled backgrounds in general... Played around with it, looks like this with our current color tokens:

<img width="302" alt="Screenshot 2025-03-21 at 08 24 06" src="https://github.com/user-attachments/assets/bf7ed85e-12f1-4067-b6a9-596395d7d44e" />

With your state:

<img width="298" alt="Screenshot 2025-03-21 at 08 25 46" src="https://github.com/user-attachments/assets/6da4b5b1-687a-4f75-98aa-2cbb9089f965" />

What do you think about that?

Reply:
eh... kind of unrelated to this PR, since you made the slider more neutral. Sorry, didn't get that before... If you think the tertiary-container might be a thing for the switch -> ping me or open an issue please. :-)